framework/widgets/ActiveField.php

```diff
@@ -145,6 +145,10 @@ class ActiveField extends Component
      * it is maintained by various methods of this class.
      */
     public $parts = [];
+    /**
+     * @var bool adds ARIA HTML attributes `aria-required` and `aria-invalid` for inputs
+     */
+    public $addAriaAttributes = false;
```

Comment:
how about enabling this by default. as far as I see it can not break anything and will make yii apps accessable by default?